cdap-ui/app/hydrator/controllers/create/partials/console-tab-ctrl.js

```diff
@@ -48,6 +48,9 @@ class HydratorPlusPlusConsoleTabService {
         case 'INVALID-CONNECTIONS':
           errorMessage.push(message.payload.connections.join(', ') + ' - invalid connection');
           break;
+        case 'NO-BACKEND-PROPS':
```

Comment:
`case` statements in `switch` can have block scope.

```
case 'NO-BACKEND-PROPS`: {
  const multiplePlugins = message.payload.nodes.length > 1;
  const suffix = multiplePlugins ? 's' : '';
  const pluginNames = message.payload.nodes.join(', ')
  const message = `Artifacts${suffix}  ${pluginNames} ${multiplePlugins ? 'are' : 'is} not available`;
  break;
}
```
Makes it slightly better to read.

Reply:
Made it more readable